scripts/au_extensions.psm1

```diff
@@ -5,6 +5,7 @@
 # but the file containing the functions is expected
 # to be named using the same name.
 $funcs = @(
+  'Get-LatestGitHubRelease'
```

Comment:
If the tag is made configurable, this should be changed to be named `Get-GitHubRelease` instead.

Reply:
This so happens to be the same naming as one of the current cmdlet from the [PowerShellForGitHub](https://github.com/microsoft/PowerShellForGitHub) module